login.html

```diff
@@ -1,11 +1,65 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓 - 회원가입</title>
+    <link rel="icon" href="images/logo/favicon.ico" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
+    <link rel="stylesheet" href="Style/login.css" />
+  </head>
+
+  <body>
+    <form>
+      <h1>
+        <a href="index.html">
+          <img id="login" src="images/logo/panda-market-logo.png" alt="pandalogin">
+        </a>
+        
+      </h1>
+      <div class="container">
+        <label class="emailpass">이메일</label>
+        <input
+          name="email"
+          type="email"
+          placeholder="codeit@email.com"
+        />  
```

Comment:
❗️ 수정요청
특정 input이 어떤 값을 받는지 설명하는 text들을 label 이라고 합니다.
이러한 친구들은 일반적으로 마우스로 선택시 연결된 input으로 focus가 이동하는 기능이 있습니다~
이를 위해, 그리고 명확한 연결관계 표시하기 위해 label 태그를 사용해주시는 것이 좋습니다~

```suggestion
        <label for="email" class="emailpass">이메일</label> // for 속성에는 연결할 Input id를 넣어주세요
        <input
          id="email"
          name="email"
          type="email"
          placeholder="codeit@email.com"
        />  
```